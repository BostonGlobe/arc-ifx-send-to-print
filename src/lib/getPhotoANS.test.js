const getPhotoANS = require('./getPhotoANS');

describe('getPhotoANS', () => {
  it('fetches the photo ANS from the photo API', async () => {
    const result = await getPhotoANS('PUWLWI4TH5EMNPZSVZPLLJT5D4');

    expect(result._id).toEqual('PUWLWI4TH5EMNPZSVZPLLJT5D4');
  });
});
