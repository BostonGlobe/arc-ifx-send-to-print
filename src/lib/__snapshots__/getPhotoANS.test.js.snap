// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getPhotoANS fetches the photo ANS from the photo API 1`] = `
{
  "_id": "PUWLWI4TH5EMNPZSVZPLLJT5D4",
  "additional_properties": {
    "fullSizeResizeUrl": "/resizer/B0YRzst8M9gZdG3aJfRwIrOVkt0=/arc-anglerfish-arc2-sandbox-sandbox-bostonglobe/public/PUWLWI4TH5EMNPZSVZPLLJT5D4.jpg",
    "galleries": [],
    "ingestionMethod": "manual",
    "iptc_job_identifier": "6716aab8b1cdf92f64b75f6b",
    "iptc_source": "X2-REMOTE TRANSMISSION",
    "iptc_title": "sox rolling rally",
    "mime_type": "image/jpeg",
    "originalName": "https___arcmigration-prdweb.bostonglobe.com_r_Boston_2011-2020_2011_08_18_BostonGlobe.com_ReceivedContent_Images_2004sox47.jpg",
    "originalUrl": "https://cloudfront-us-east-1.images.arcpublishing.com/sandbox.bostonglobe/PUWLWI4TH5EMNPZSVZPLLJT5D4.jpg",
    "owner": "ryan.huddle@globe.com",
    "proxyUrl": "/resizer/B0YRzst8M9gZdG3aJfRwIrOVkt0=/arc-anglerfish-arc2-sandbox-sandbox-bostonglobe/public/PUWLWI4TH5EMNPZSVZPLLJT5D4.jpg",
    "published": true,
    "resizeUrl": "/resizer/B0YRzst8M9gZdG3aJfRwIrOVkt0=/arc-anglerfish-arc2-sandbox-sandbox-bostonglobe/public/PUWLWI4TH5EMNPZSVZPLLJT5D4.jpg",
    "restricted": false,
    "takenOn": "2004-10-30T00:00:00Z",
    "template_id": 687,
    "thumbnailResizeUrl": "/resizer/J83FXXRAul1ot-ihuUlqMegQbIg=/300x0/arc-anglerfish-arc2-sandbox-sandbox-bostonglobe/public/PUWLWI4TH5EMNPZSVZPLLJT5D4.jpg",
    "version": 1,
  },
  "address": {
    "locality": "Boston",
    "region": "MA",
  },
  "auth": {
    "1": "8aa0cea418800c48709b6779354eba2e798a35b6686cbca340530a9a24c06539",
  },
  "caption": "Red Sox Rolling Rally-BOSTON- A Red Sox fan walks  into the Charles River to try to get to duck carrying Manny Ramirez, David Ortiz and others RED SOX PARADE ROLLING RALLY WORLD SERIES          OUTTAKe",
  "created_date": "2024-10-24T15:03:53Z",
  "credits": {
    "affiliation": [],
    "by": [
      {
        "byline": "Stan Grossfeld/Globe Staff",
        "name": "Stan Grossfeld/Globe Staff",
        "type": "author",
      },
    ],
  },
  "distributor": {
    "mode": "reference",
    "reference_id": "5c6729d7-3c7c-41ac-9ae7-f437c0c6c6f3",
  },
  "height": 1824,
  "image_type": "photograph",
  "last_updated_date": "2024-10-24T15:04:17Z",
  "licensable": false,
  "owner": {
    "id": "sandbox.bostonglobe",
    "sponsored": false,
  },
  "planning": {
    "internal_note": "globe staff template",
  },
  "slug": "1.0.1283556784",
  "source": {
    "additional_properties": {
      "editor": "photo center",
    },
    "edit_url": "https://sandbox.bostonglobe.arcpublishing.com/photo/PUWLWI4TH5EMNPZSVZPLLJT5D4",
    "name": "Boston Globe",
    "source_id": "The Boston Globe",
    "source_type": "staff",
    "system": "photo center",
  },
  "subtitle": "sox rolling rally",
  "syndication": {},
  "taxonomy": {
    "associated_tasks": [
      "6716aab8b1cdf92f64b75f6b",
    ],
  },
  "type": "image",
  "url": "https://cloudfront-us-east-1.images.arcpublishing.com/sandbox.bostonglobe/PUWLWI4TH5EMNPZSVZPLLJT5D4.jpg",
  "version": "0.10.9",
  "width": 2922,
}
`;
